src/main/java/yandex/mail/pages/InboxPage.java

```diff
@@ -0,0 +1,22 @@
+package yandex.mail.pages;
+
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import java.time.Duration;
+import java.util.NoSuchElementException;
+
+import static yandex.mail.Locators.INBOX_TITLE;
+
+public class InboxPage extends BasePage {
+
+    public String getTitle() {
+        new FluentWait(driver)
+                .withTimeout(Duration.ofSeconds(15))
+                .pollingEvery(Duration.ofSeconds(3))
+                .ignoring(NoSuchElementException.class)
+                .until(ExpectedConditions.urlContains("?uid=1876203154#tabs/relevant"));
```

Comment:
"?uid=1876203154#tabs/relevant" - for now it's ok, but usually don't use specific user id - often tests uses different users and it would fail